website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,441 @@
+
+#### Bonus: Android app size reduction
+
+The minimum SDK bump, together with several improvements at our native build, allowed us to greatly reduce the app size on user devices.
+
+For example a newly created app with React Native 0.74 occupies ~13% less space on user device, resulting in ~4Mb saved on device (see the attached screenshot).
+
+### Yarn 3 for New Projects
+
+[Yarn 3](https://yarnpkg.com/blog/release/3.0) is now the default JavaScript package manager for new projects initialized with React Native Community CLI.
+
+Yarn 3.x will be used with `nodeLinker: node-modules`, a mode providing compatibility with React Native libraries. This replaces Yarn Classic (1.x, deprecated) as the previous default. To upgrade Yarn version inside your existing app you can follow this [guide](https://yarnpkg.com/migration/guide).
+
+```sh
+$ yarn --help
+━━━ Yarn Package Manager - 3.6.4 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  $ yarn <command>
+```
+
+The Community CLI also supports initializing projects with other package managers via the `--pm` flag ([read more](https://github.com/react-native-community/cli/blob/main/docs/init.md)).
+
+## Breaking Changes
+
+### Removal of deprecated `PropTypes`
+
+Before 0.74, React Native was continuing to ship with `PropTypes`, an API that had been deprecated since React 15.5 in 2017! The presence of this deprecated code incurs costs in app size and memory overhead for apps that follow best practices.
+
+Starting in 0.74, React Native will no longer include the following `PropTypes` properties: `Image.propTypes`, `Text.propTypes`, `TextInput.propTypes`, `ColorPropType`, `EdgeInsetsPropType`, `PointPropType`, `ViewPropTypes` (see [commit](https://github.com/facebook/react-native/commit/228cb80af9ded20107f3c7a30ffe00e24471bfeb)).
+
+If your app or library relies on `PropTypes`, we highly recommend migrating to a type system like TypeScript. If you have trouble identifying the source of a deleted `PropTypes` usage, please [file an issue](https://github.com/facebook/react-native/issues) so we can help track it down with you.
+
+### API Changes to PushNotificationIOS (Deprecated)
+
+In React Native 0.74, we are making steps to remove the deprecated [PushNotificationIOS](https://reactnative.dev/docs/pushnotificationios) library. The changes in this release are focused on removing references to older iOS APIs. PushNotificationIOS has been migrated onto Apple’s [User Notifications](https://developer.apple.com/documentation/usernotifications?language=objc) framework and exposes new APIs for scheduling and handling notifications.
+
+In the next release (0.75), **we are planning to remove this library**, relocating it out of React Native core and into the community package, [@react-native-community/push-notification-ios](https://github.com/react-native-push-notification/ios). If you are still relying on PushNotificationIOS, you’ll need to migrate over before the next release.
```

Comment:
I've reduced the bold *now only the substring "remove this library"). I think it's worth having this stand out to signal this major intended change, which is not otherwise signalled in the subheading.